src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,7 @@
+package wolox.training.exceptions;
+
+public class BookIdMismatchException extends RuntimeException{
```

Comment:
Lo recomendable para evitar dejar los mensajes de errores en el controller, es que se agregue la anotación @ResponseStatus(code = HttpStatus. BAD_REQUEST, reason = "id is different to id in the object Book") en la clase junto con el code y reason de la exception.